src/ucp/rndv/rndv.c

```diff
@@ -578,25 +555,24 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_progress_rma_get_zcopy, (self),
     }
 }
 
-UCS_PROFILE_FUNC_VOID(ucp_rndv_get_completion, (self),
-                      uct_completion_t *self)
+UCS_PROFILE_FUNC_VOID(ucp_rndv_get_completion, (self), uct_completion_t *self)
 {
     ucp_request_t *rndv_req = ucs_container_of(self, ucp_request_t,
                                                send.state.uct_comp);
 
+    uct_completion_update_status(&rndv_req->send.state.uct_comp, self->status);
```

Comment:
AFAIR `ucp_rndv_get_completion` is already called from UCT level, so status should be up to date. `self == &rndv_req->send.state.uct_comp`, right?

Reply:
good catch
removed